Games endpoints

hjwk opened the pull request:
For adding games within an event

waliens commented:
Hmm, not sure about that. Not sure access rights are verified at all with the `AddGame` endpoint (?)

hjwk commented:
> IMO, keeping both makes sense as both have different access checks (although it can be discussed given we have deleted `EditEventGame`)

The reason I deleted the addEventGame route and code was because I thought it was a leftover of the modfications you made when you deleted EditEventGame. I thought that we had agreed in the unificiation of the endpoints for game creation/update but looking back on the PR it wasn't clearly stated that it also concerned the creation.

waliens commented:
If we choose to keep both routes, I think it is better to drop the PR instead of reverting the state. What do you think ?

hjwk commented:
This PR still solves an issue with the creation of games

waliens commented:
Oops sorry I had missed that, I didn't see the `req.params.gid` thing. 
However, I think the other modifications should not be done (except the import cleaning). The `addGame` function had the purpose of passing `null` instead of `undefined` to the handler function where `null` is actually the expected value for the event id while `undefined` is not.
